packages/gestalt/src/Popover.js

```diff
@@ -9,18 +9,69 @@ type Size = 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'flexible' | number;
 type IdealDirection = 'up' | 'right' | 'down' | 'left';
 
 type Props = {|
+  /**
+   * The reference element, typically [Button](/Button) or [IconButton](/IconButton), that Popover uses to set its position.
```

Comment:
@AlbertCarreras could you make sure to add full URLs? These docs are also used by Flow and otherwise won't work inside of our IDE's.

https://github.com/pinterest/gestalt/blob/4698948d06bc6313084cd0a8ba6ecb1c3eff9df6/packages/gestalt/src/Modal.js#L28

So instead:

```
The reference element, typically [Button](https://gestalt.pinterest.systems/Button) or [IconButton](https://gestalt.pinterest.systems/IconButton), that Popover uses to set its position.
```

Reply:
Is that something we can configure at a higher level? Add an assumption that all relative links are relative from a certain path?

Reply:
@dangerismycat not at the moment.

A potential way to fix this would be to add the base URL `https://gestalt.pinterest.systems/` for relative links when we publish. However, it breaks if we want to test out these links within VSCode while developing gestalt. So for now, I would suggest to keep adding `https://gestalt.pinterest.systems/` to all urls in these docs.